### PR TITLE
DM-27865: make the code like DM-27757 on the main line, including restoring CCW tracking enable/disable.

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: bin/tma_commander.py [hunk 1]
```diff
@@ -130,31 +130,18 @@ def __init__(self, host, log_level, simulate):
         )
         self.command_loop_task = asyncio.create_task(self.command_loop())
         self.command_dict = dict(
-            az_power=MTMount.commands.AzimuthAxisPower,
```
Comment (tribeiro) [inline bin/tma_commander.py:133]: Are you sure you want to remove these commands? I think it is ok to keep them around in the commander, gives us a chance that check what happens if we issue them.

**Review by tribeiro — APPROVED** [review]
Looks good.

File: bin/tma_commander.py (hunk 1 above)
Comment (r-owen) [reply bin/tma_commander.py:133]: I'm confused. Why would we want to try to send commands to elevation and azimuth? I guess it's not actively harmful, but it seems strange.

Comment (r-owen) [reply bin/tma_commander.py:133]: I'll restore all commands. I guess it's a bit simpler having the file identical to the main branch.

**Pull request merged**